Prevent 502 errors when using AJAX

mattkingshott opened the pull request:
When using the Laravel DebugBar with the `capture_ajax` flag set to `true`, developers may encounter 502 Bad Gateway errors. This is due to DebugBar's AJAX request monitoring exceeding the FastCGI buffer limit.

This PR increases the size of the buffer. While it is not guaranteed to fix the problem, the increased buffer size should solve the issue for most developers. Further discussion on this subject can be [found in this issue on the DebugBar repo](https://github.com/barryvdh/laravel-debugbar/issues/251).

drbyte commented:
Is it intentional that this only applies to unsecured sites? ie: only on sites for which `valet secure` has NOT been run?

The change you're proposing is in the config file used only for unsecured sites. 
If one wishes this to be applied to both secured and unsecured sites, note that both types of site stubs have an `include fastcgi_params` directive, meaning the proposed changes could be placed in that file instead.

Also, is the temp_file_write_size and busy_buffers part really needed?  Forcing unneeded settings on every Valet user can lead to unexpected outcomes.

mattkingshott commented:
Thanks for the comments @drbyte. It wasn't intentional no, so I've done the following...

I've moved the required parameters to `fastcgi_params` and removed the unnecessary ones.

drbyte commented:
Wondering: How does doing this affect RAM demands on sites?  eg: what are the new requirements for using Valet?

mattkingshott commented:
I’ve no idea to be honest, but is it likely to be an issue given that Valet is a dev environment and not for use in production?

Either way, I’ve not noticed any difference with it enabled or not.

mattstauffer commented:
There are a lot of people in [this thread](https://github.com/barryvdh/laravel-debugbar/issues/2510) either talking about this or emoji responding positively to the folks talking about it, so I'm going to consider it tested enough, and merge it. Thanks!
